script.js

```diff
@@ -1,4 +1,184 @@
+    if (playerThrow[0] == playerThrow[1]) {
+      whichDiceOrder = 0;
+      turnOrOrder += 1;
+      //result for P1
+      if (turnOrOrder == 2) {
+        pOneResult = "" + playerThrow[0] + playerThrow[1];
+        //result for P2
+        myOutputValue =
+          myOutputValue +
+          `<br><br>Your hand is ${playerThrow[0]}${playerThrow[1]}.`;
+        return myOutputValue;
+      } else if (turnOrOrder == 4) {
+        pTwoResult = "" + playerThrow[0] + playerThrow[1];
+        myOutputValue =
+          myOutputValue +
+          `<br><br>Your hand is ${playerThrow[0]}${playerThrow[1]}.`;
+        return myOutputValue;
+      }
+    }
+
+    //Else ask player which order they want to choose
+    myOutputValue =
+      myOutputValue +
+      ` <br><br>Which order would you like to put them?<br> 1) die 1 first : ${playerThrow[0]}${playerThrow[1]} <br> 2) die 2 first : ${playerThrow[1]}${playerThrow[0]}`;
+    return myOutputValue;
+  } // Player to choose order of die:
+  else if (turnOrOrder == 1 && whichDiceOrder == 1) {
+    // Player 1 return message (die 1 first)
+    if (input == 1) {
+      turnOrOrder += 1;
+      whichDiceOrder = 0;
+      pOneResult = "" + playerThrow[0] + playerThrow[1];
+      return `Ok, your hand is ${pOneResult}. <br><br>It is now Player 2's turn.`;
+    } // player 1 return message (die 2 first)
+    else if (input == 2) {
+      turnOrOrder += 1;
+      whichDiceOrder = 0;
+      pOneResult = "" + playerThrow[1] + playerThrow[0];
+      return `Ok, your hand is ${pOneResult}. <br><br>It is now Player 2's turn.`;
+    } else {
+      return `Please select a valid number.<br><br>${myOutputValue}`;
+    }
+  } // Player 2 choose order of die
+  else if (turnOrOrder == 3 && whichDiceOrder == 1) {
+    if (input == 1) {
+      console.log("p2 1");
+      turnOrOrder = 4;
+      whichDiceOrder = 0;
+      pTwoResult = "" + playerThrow[0] + playerThrow[1];
+      myOutputValue = `Ok, your hand is ${pTwoResult}.`;
+      return myOutputValue;
+    } else if (input == 2) {
+      console.log("p2 2");
+      turnOrOrder = 4;
+      whichDiceOrder = 0;
+      pTwoResult = "" + playerThrow[1] + playerThrow[0];
+      myOutputValue = `Ok, your hand is ${pTwoResult}`;
+      return myOutputValue;
+    } else {
+      return `Please select a valid number.<br><br>${myOutputValue}`;
+    }
+  }
+
+  // Evaluate results for winner
+  else if (turnOrOrder == 4 && whichDiceOrder == 0) {
+    turnOrOrder = 0;
+    pOneStats = Math.floor(pOneStats) + Math.floor(pOneResult);
+    pTwoStats = Math.floor(pTwoStats) + Math.floor(pTwoResult);
+    gameStats();
+    //P2 results bigger
+    // add in game mode condition
+    if (
+      (pOneResult > pTwoResult && gameMode == 1) ||
+      (pOneResult < pTwoResult && gameMode == 2)
+    ) {
+      myOutputValue =
+        myOutputValue +
+        ` but Player 1's hand is ${pOneResult}. You lost :(<br><br>The game will now restart!` +
```

Comment:
seems like the output on who wins/loses is always from the pov of player 2, was this design choice intentional? small issue regardless, but personal preference would be for the output to say specifically which player won